src/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -106,9 +106,13 @@ void ShootGoalTactic::shootUntilShotBlocked(std::shared_ptr<KickAction> kick_act
                                             std::shared_ptr<ChipAction> chip_action,
                                             ActionCoroutine::push_type &yield) const
 {
-    auto shot_target = Evaluation::calcBestShotOnEnemyGoal(field, friendly_team,
-                                                           enemy_team, ball.position());
-    while (shot_target)
+    std::optional<Shot> shot_target = Evaluation::calcBestShotOnEnemyGoal(
+        field, friendly_team, enemy_team, ball.position(), ROBOT_MAX_RADIUS_METERS,
+        {*this->getAssignedRobot()});
+
+
+
```

Comment:
nit, can we remove two of three blank lines here?

Reply:
Removed